config/sync/field.field.node.stanford_page.su_basic_page_tags_filters.yml

```diff
@@ -0,0 +1,29 @@
+uuid: 34e6e613-d839-4ca2-b5b9-51bc59ce2e7e
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.su_basic_page_tags_filters
+    - node.type.stanford_page
+    - taxonomy.vocabulary.basic_page_filters
+id: node.stanford_page.su_basic_page_tags_filters
+field_name: su_basic_page_tags_filters
+entity_type: node
+bundle: stanford_page
+label: 'Basic Page Tags/Filters'
+description: 'Enter the categories associated with this basic page. These will be used in two ways: They will be displayed in the sidebar on the individual basic page. They will also be used as filters for the Filtered Lists Paragraph view.'
```

Comment:
@mdyoung3 Let's change this to ""These will be used as filters for the future Filtered Lists Paragraph for Basic Page. They do not display on the Basic Page or Basic Page Teasers."